website/.nvmrc

```
@@ -0,0 +1 @@
+v16.17.0
```

```suggestion
lts/*
```

This should ensure we're on latest stable and saves us the trouble with maintaining this version.
Node.js LTS update rarely break things

---

Or maybe `v16` to get security updates and bug fixes

---

Agreed, updated to v16 👍 We can upgrade major versions manually